exec.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+docker build -t auth0-publish-rubygem .
+docker run -it auth0-publish-rubygem
```

Comment:
Will the container interactive shell close after running the "default command"? and in that case, shouldn't you add the `--rm` flag to also shutdown the container?